src/NuGet.Clients/NuGet.VisualStudio.Common/Services/IPMUIStarter.cs

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace NuGet.VisualStudio
+{
+    public interface IPMUIStarter
+    {
+        void PMUIStarter();
```

Comment:
```suggestion
        void StartPMUI();
```

Reply:
I'd prefer spelling out PMUI. Something like:
`LaunchPackageManager` and if it's specifically solution PMUI:
`LaunchSolutionPackageManager`

Or replace Launch with Show.

Reply:
Even better :)